### PR TITLE
refactor: transition heading buttons

### PR DESCRIPTION
**Pull request by whitehat**
## Proposed changes

Add Transition To Buttons On Heading

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/components/header/ToggleChart.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<template>
+ transition<template>
```
Comment (dated) [inline src/components/header/ToggleChart.vue]: ?

**Review by dated — CHANGES_REQUESTED** [review]

File: src/components/header/ToggleChart.vue (hunk 1 above)
Comment (whitehat) [reply src/components/header/ToggleChart.vue]: Sorry. Thought Fixed Before Commit. Deleted Now

Comment (dated) [reply src/components/header/ToggleChart.vue]: no problem :D

**Review by dated — DISMISSED** [review]

**Comment by whitehat** [comment]
Ugh The Space Too, Sorry :woman_facepalming:

**Review by dated — APPROVED** [review]

**Pull request merged**